### DISCUSSION TITLE
References

### DISCUSSION THREAD

**Discussion by CommonClimate** [alshedivat/al-folio#415] [Q&A] [answered]
Hello al-folio community,
I am trying to put 2 kinds of references on my page and can't seem to find how:

1.  relative links to other pages, e.g. linking from a news item to a project using a link of the form `[check out this project](../_projects/myproject.markdown), which does not work ("Not Found" error). Does anyone know a workaround?
2. On project pages, I'd like to incorporate references to a few relevant publications, in the same way that the landing page includes references to selected publications.  It seems straightforward to add a "project" field to the .bib items, but how would I get the relevant project page to display those citations? Or indeed, how can I cite references from the bib file, generally speaking?

Sorry if these questions are too basic! Hopefully there is an easy answer our there. 
Thanks in advance for any tips you may provide.
Julien

**Comment by RemyDenechere** [comment]
it is explained somewere in one of the projects: 
"_projects/7_project.md:You can also put regular text between your rows of images, even citations {% cite einstein1950meaning %}."

**Comment by george-gca** [comment, marked as answer]
Actually this question is really old, and the references part didn't exist by then.
Currently there is an example in the [Project 1](https://alshedivat.github.io/al-folio/projects/1_project/):

1. Enable citation

https://github.com/alshedivat/al-folio/blob/8121ad02690c256b997a4b068714d909d1db796d/_projects/1_project.md?plain=1#L8

2. Cite

https://github.com/alshedivat/al-folio/blob/8121ad02690c256b997a4b068714d909d1db796d/_projects/1_project.md?plain=1#L47

About linking to other pages, you can use the [link or post_url](https://jekyllrb.com/docs/liquid/tags/#links) tags, depending on your situation. Also, you can use something like `{{ url | relative_url }}`. You can see how it works in the [docs](https://jekyllrb.com/docs/liquid/filters/).

If nothing else works, you can try [jekyll-wikirefs](https://github.com/wikibonsai/jekyll-wikirefs) plugin (I haven't tried it myself).
